tests/foreman/cli/test_errata.py

```diff
@@ -474,12 +1142,63 @@ def test_positive_list_filter_by_product_name_and_org_id(self):
         1. erratum list --product=<product_name> --organization-id=<org_id>
 
         @Assert: Errata is filtered by product name and Org id.
-
-        @caseautomation: notautomated
-
         """
-
-    @stubbed()
+        product_erratum_list = Erratum.list({
```

Comment:
I compared the assert part of test bellow this line and it is exactly the same as the on begining on line 1085. I suppose it is the same on other tests that I made comment about set with empty list. So what you think about extraction a method `assert_errata_conditions' receiving the all errata related as parameters? At least we would have less code and no repetition.